tools/convert/conversions/1.0.2-1.1.json

```diff
@@ -0,0 +1,11995 @@
+    },
+    {
+      "description": "Add brick:Discharge_Air_Temperature_Heating_Setpoint",
+      "query": "INSERT{ ?subject ?predicate brick:Discharge_Air_Temperature_Heating_Setpoint . } WHERE { ?subject  ?predicate  brick_v_1_0_2:AHU_Discharge_Air_Temperature_Heating_Setpoint . }"
+    },
+    {
+      "description": "Remove brick:AHU_Discharge_Air_Temperature_Heating_Setpoint",
+      "query": "DELETE{ ?subject ?predicate brick_v_1_0_2:AHU_Discharge_Air_Temperature_Heating_Setpoint . } WHERE { ?subject  ?predicate  brick_v_1_0_2:AHU_Discharge_Air_Temperature_Heating_Setpoint . }"
+    },
+    {
+      "description": "Add brick:Discharge_Air_Temperature_Sensor",
+      "query": "INSERT{ ?subject ?predicate brick:Discharge_Air_Temperature_Sensor . } WHERE { ?subject  ?predicate  brick_v_1_0_2:AHU_Discharge_Air_Temperature_Sensor . }"
+    },
+    {
+      "description": "Remove brick:AHU_Discharge_Air_Temperature_Sensor",
+      "query": "DELETE{ ?subject ?predicate brick_v_1_0_2:AHU_Discharge_Air_Temperature_Sensor . } WHERE { ?subject  ?predicate  brick_v_1_0_2:AHU_Discharge_Air_Temperature_Sensor . }"
+    },
+    {
+      "description": "Add brick:Discharge_Air_Temperature_Setpoint",
+      "query": "INSERT{ ?subject ?predicate brick:Discharge_Air_Temperature_Setpoint . } WHERE { ?subject  ?predicate  brick_v_1_0_2:AHU_Discharge_Air_Temperature_Setpoint . }"
+    },
+    {
+      "description": "Remove brick:AHU_Discharge_Air_Temperature_Setpoint",
+      "query": "DELETE{ ?subject ?predicate brick_v_1_0_2:AHU_Discharge_Air_Temperature_Setpoint . } WHERE { ?subject  ?predicate  brick_v_1_0_2:AHU_Discharge_Air_Temperature_Setpoint . }"
+    },
+    {
+      "description": "Add brick:Command",
+      "query": "INSERT{ ?subject ?predicate brick:Command . } WHERE { ?subject  ?predicate  brick_v_1_0_2:AHU_Discharge_Fan_Command . }"
+    },
+    {
+      "description": "Remove brick:AHU_Discharge_Fan_Command",
+      "query": "DELETE{ ?subject ?predicate brick_v_1_0_2:AHU_Discharge_Fan_Command . } WHERE { ?subject  ?predicate  brick_v_1_0_2:AHU_Discharge_Fan_Command . }"
+    },
+    {
+      "description": "Add brick:Start_Stop_Command",
+      "query": "INSERT{ ?subject ?predicate brick:Start_Stop_Command . } WHERE { ?subject  ?predicate  brick_v_1_0_2:AHU_Discharge_Fan_Start_Stop_Command . }"
+    },
+    {
+      "description": "Remove brick:AHU_Discharge_Fan_Start_Stop_Command",
+      "query": "DELETE{ ?subject ?predicate brick_v_1_0_2:AHU_Discharge_Fan_Start_Stop_Command . } WHERE { ?subject  ?predicate  brick_v_1_0_2:AHU_Discharge_Fan_Start_Stop_Command . }"
+    },
+    {
+      "description": "Add brick:Start_Stop_Status",
+      "query": "INSERT{ ?subject ?predicate brick:Start_Stop_Status . } WHERE { ?subject  ?predicate  brick_v_1_0_2:AHU_Discharge_Fan_Start_Stop_Status . }"
+    },
+    {
+      "description": "Remove brick:AHU_Discharge_Fan_Start_Stop_Status",
```

Comment:
@shreyasnagare Sorry, please correct me if I am wrong. It is removing `AHU_Discharge_Fan_Start_Stop_Status` but not replacing it with `Start_Stop_Status`. Could you point out where you add that?

Reply:
nvm my bad. It's right above the line.

Reply:
> @shreyasnagare Sorry, please correct me if I am wrong. It is removing `AHU_Discharge_Fan_Start_Stop_Status` but not replacing it with `Start_Stop_Status`. Could you point out where you add that?

The query right before this one adds that.